prov/sockets/src/sock_fabric.c

```diff
@@ -580,19 +581,29 @@ SOCKETS_INI
 {
 	char *value;
 
-	if ((value = getenv("SOCK_PE_WAITTIME"))) {
+	if (fi_var_register(&sock_prov, "pe_waittime",
+				"How many miliseconds to spin while waiting for progress"))
+		SOCK_LOG_INFO("Failed to register pe_waittime variable\n");
+	if (fi_var_get(&sock_prov, "pe_waittime", &value))
+		SOCK_LOG_INFO("Failed to read pe_waittime variable\n");
```

Comment:
Since the var is now FI_SOCKETS_*, should the log report the same? The name of the var is not consistent in the logs.